ngraph/python/src/pyngraph/node.cpp

```diff
@@ -276,4 +276,10 @@ void regclass_pyngraph_Node(py::module m) {
         util::DictAttributeDeserializer dict_deserializer(attr_dict, variables);
         self->visit_attributes(dict_deserializer);
     });
+    node.def("_set_arguments", [](const std::shared_ptr<ngraph::Node>& self, const ngraph::OutputVector& arguments) {
```

Comment:
This should be `set_arguments` since it is going to be a public a public API from now on. Same applies for `_set_attribute` and `_get_attributes` in my opinion. As well we can provide single value return `get_attribute` function and setting of multiple a attributes with `set_attributes` which allows to pass py::dict as argument and iterate over it.

Reply:
@sdurawa also `attributes` getter and setter should be public since we can set them by hand.

Reply:
Fixed.